lib/bundle/dumper.rb

```diff
@@ -20,6 +20,7 @@ def self.dump_brewfile
       content << cask_before_formula
       content << BrewDumper.dump
       content << cask_after_formula
+      content << MacAppStoreDumper.dump
```

Comment:
It will already not dump anything unless `mas` is installed. I think this argument about Mac Apps applies equally to brews, casks and taps. You can always edit your `Brewfile` after dumping if you have `mas` installed but don't want to use it for some reason. I don't think an option is useful unless it defaults to not dumping at which point this feature is not discoverable or useful. If you'd like to add an option in a follow-up PR for personal usage: that's fine.